resources/assets/coffee/react/beatmapset-page/main.coffee

```diff
@@ -74,13 +74,6 @@ export class Main extends React.Component
   }) =>
     @scoreboardXhr?.abort()
 
-    @setState
-      currentScoreboardType: scoreboardType
```

Comment:
this should stay otherwise it's annoying not seeing the option change after clicking.

If anything, this should also set enabled mods; save previous type and mods to a variable beforehand and restore it on fail.

Reply:
This cause flickering when fail fast enough.

![flickmod](https://user-images.githubusercontent.com/4964985/123083485-4aa61980-d44a-11eb-8513-58f4a5ef23b9.gif)

Reply:
that seems like a very rare case or just dead internet

Reply:
Okay, I just pushed it.